src/components/Home/index.js

```diff
@@ -84,15 +97,15 @@ const ChoresList = ({ chores }) => (
         <ul className="listFrame">
             <h2 className="homeTitle">
                 <Link className="homeTitle" to={ROUTES.CHORES}>Chores</Link>
-                <button className="addButtonFrame"><Add className="addButton">A</Add></button>
+                <button className="addButtonFrame" data-toggle="modal" data-target="#Chores"><Add className="addButton"></Add></button>
             </h2>
             {chores.map((chore) => (
                 <div className="card itemFrame mt-1">
                     <div className="card-body ">
                         <li key={chore.Timestamp}>
                             <div className="item">
                                 <button type="button" className="options btn btn-primary dropdown-toggle" id="dropdownOptions" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Options</button>
-                                <div class="dropdown-menu" aria-labelledby="dropdownOptions">
+                                <div className="dropdown-menu" aria-labelledby="dropdownOptions">
                                     <button className="dropdown-item">Complete</button>
                                     <button className="dropdown-item">Edit</button>
                                     <button className="dropdown-item">Add to Calender</button>
```

Comment:
Should add a remove option.

Reply:
We have "Complete" item in there as we don't want it to be removed from the database, but rewording might be necessary.